seed/data/notificationTemplate/notificationTemplates.js

```diff
@@ -0,0 +1,23 @@
+module.exports = [
```

Comment:
These should not be here. They will be managed separately by the other process you set up.

Reply:
done